src/main/java/htsjdk/variant/variantcontext/JEXLMap.java

```diff
@@ -5,61 +5,66 @@
 import org.apache.commons.jexl2.JexlException;
 import org.apache.commons.jexl2.MapContext;
 
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
 
 /**
  * This is an implementation of a Map of {@link JexlVCMatchExp} to true or false values.
  * It lazily initializes each value as requested to save as much processing time as possible.
  */
 
 class JEXLMap implements Map<JexlVCMatchExp, Boolean> {
```

Comment:
This class `JEXLMap` appears to have no direct unit testing at all. Can you add a `JEXLMapUnitTest` that at least covers the three different missing value behaviors?